webpack/assets/javascripts/react_app/components/Template/Inputs/AutoComplete.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FieldLevelHelp } from 'patternfly-react';
+
+import Form from '../../common/forms/CommonForm';
+import AutoComplete from '../../AutoComplete';
+
+const TemplateAutoComplete = ({
+  label,
+  resourceType,
+  id,
+  initialQuery,
+  initialError,
+  info,
+  isRequired,
+  url,
+  template,
+}) => (
+  <Form
+    touched
+    label={label}
+    error={initialError}
+    required={isRequired}
+    tooltipHelp={
+      info && (
+        <FieldLevelHelp
+          buttonClass="field-help"
+          content={<Fragment>{info}</Fragment>}
+        />
+      )
+    }
+  >
+    <AutoComplete
+      id={`template-autocomplete-input-${id}`}
+      inputProps={{
+        name: `${template}[input_values][${id}][value]`,
+      }}
+      controller={resourceType}
+      url={url}
+      initialQuery={initialQuery}
+      error={initialError}
```

Comment:
It's better to use `initialError` prop now

Reply:
Thanks, actually I removed it, because the wrapper form handles validation errors